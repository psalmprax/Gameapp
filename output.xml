<?xml version="1.0" encoding="UTF-8" standalone="no"?><route><room id="1" name="Stairway"/><room id="2" name="Hallway 1"><object name="Scarf"/><object name="Umbrella"/></room><room id="4" name="Storage Room"/><room id="2" name="Hallway 1"/><room id="3" name="Hallway 2"/><room id="5" name="Kitchen"><object name="Knife"/></room></route>