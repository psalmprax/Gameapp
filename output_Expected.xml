<?xml version="1.0" encoding="UTF-8" standalone="no"?><route><room id="1" name="Stairway"/><room id="2" name="Hallway 1"><object name="Scarf"/><object name="Soap"/></room></route>
